helpers/emulation.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+// Nexus5X metrics adaparted from emulated_devices/module.json
+const DEVICE_EMULATION_METRICS = {
+  mobile: true,
+  screenWidth: 412,
+  screenHeight: 732,
+  width: 412,
+  height: 732,
+  positionX: 0,
+  positionY: 0,
+  scale: 1,
+  deviceScaleFactor: 2.625,
+  fitWindow: false,
+  screenOrientation: {
+    angle: 0,
+    type: 'portraitPrimary'
+  }
+};
+
+const DEVICE_EMULATION_USERAGENT = {
```

Comment:
I hope so, though devtools project hasnt found a good way, yet. :)

For us, we could do something along the lines of pinging the omahaproxy API every week